src/Orleans.Runtime/GrainTypeManager/GenericGrainTypeData.cs

```diff
@@ -1,27 +0,0 @@
-using System;
-using System.Reflection;
-
-namespace Orleans.Runtime
-{
-    [Serializable]
-    internal class GenericGrainTypeData : GrainTypeData
```

Comment:
why was this serializable? I assume it was an oversight and wasn't really being used as that. But just a heads up in case it was really being sent over the wire. In that case the change wouldn't be back compat.

Reply:
I think marking these as serializable was an oversight. I removed the attribute from the base class.

My plan is to move to a more stringly/open-ended format for exchanging type & other metadata for 4.0.